src/helpers/colorTemperature.js

```diff
@@ -2,6 +2,34 @@ import Geolocation from '@react-native-community/geolocation';
 
 const baseUrl = 'https://api.weather.gov/';
 
-const getColorTempGradient = () => {};
+const getColorTempGradient = temp => {
+  var textColor = '';
 
-module.exports = {getColorTempGradient};
+  //cools
+  if (temp < 0) {
+    textColor = '#00FFFF';
+  } else if (temp > 0 < 20) {
+    textColor = '#00CCFF';
+  } else if (temp > 21 < 32) {
+    textColor = '#0099FF';
+  } else if (temp > 33 < 55) {
+    textColor = '#009999';
+  }
+
+  //warms
+  else if (temp > 56 < 75) {
+    textColor = '#FFFF66';
+  } else if (temp > 76 < 85) {
+    textColor = '#FFCC66';
+  } else if (temp > 86 < 100) {
+    textColor = '#FF9966';
+  } else {
+    textColor = '#FF6666';
+  }
```

Comment:
Also for all these if else if statements we can actually utilize case statements. It’ll read better